src/test/java/by/onliner/tests/CatalogTests.java

```diff
@@ -0,0 +1,27 @@
+package by.onliner.tests;
+
+import by.onliner.AbstractTest;
+import by.onliner.core.anotation.TestType;
+import io.qameta.allure.Feature;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+import static by.onliner.constants.OnlinerConstants.IPHONE;
+import static by.onliner.constants.TestType.SMOKE;
+import static by.onliner.constants.TestType.REGRESSION;
+import static by.onliner.core.utils.EqualsUtil.equalContains;
+
+@Feature("Catalog")
+public class CatalogTests extends AbstractTest {
+
+    @TestType(value = {SMOKE, REGRESSION})
+    @Test(description = "[Test-Case ID:ONL_004] Test for finding item by searcher")
+    public void findItemBySearcher() {
+        mainPage.openPage();
```

Comment:
Also why don't you reuse stepDefinitions for the tests? why do you still use POs even though you have implemented steps